tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
Interesting...  I'm on vacation this week...  I can't see the issue without debugging...  Looking at the isValid function, I assume this.get('value') is returning NaN?  The box should still be focused and get into my change?  I'll try and look into it, but most likely will be next week before I get to it.  Sorry...